plugins/woocommerce/includes/class-wc-shipping.php

```diff
@@ -410,4 +410,27 @@ public function sort_shipping_methods() {
 		wc_deprecated_function( 'sort_shipping_methods', '2.6' );
 		return $this->shipping_methods;
 	}
+
+	/**
+	 * Check if legacy local pickup is activated in any of the shipping zones or in the Rest of the World zone.
+	 *
```

Comment:
Since this is a new public method should we consider adding a `@since` tag?

Reply:
Adding a `@since` tag, clearly doesn't harm. I've added it and set it to `8.8.0`.